tests/integration/ganache/test_compute_flow.py

```diff
@@ -390,7 +390,7 @@ def test_compute_raw_algo(
 
 
 @pytest.mark.integration
-@pytest.mark.skip(reason="Don't skip, once fixed #988")
+# @pytest.mark.skip(reason="Don't skip, once fixed #988")
```

Comment:
Did you mean to un-skip these tests? If yes, why not simply delete the whole line?